src/sbm_model.jl

```diff
@@ -438,6 +439,22 @@ function update_after_subsurfaceflow(
     return model
 end
 
+function update_total_water_storage(
```

Comment:
Same as above (but can be a lot shorter): would be nice to add a brief docstring here

Reply:
I'll add that.